x-pack/plugins/upgrade_assistant/public/application/lib/utils.ts

```diff
@@ -19,3 +21,11 @@ export const validateRegExpString = (s: string) =>
       () => ''
     )
   );
+
+export const getDeprecationsUpperLimit = (count: number) => {
```

Comment:
Can we add a comment here to explain the intention? Why don't we want to render a number greater than `DEPRECATION_WARNING_UPPER_LIMIT`?